src/components/JobCard.js

```diff
@@ -319,7 +321,10 @@ const JobCard = ({
                   </Grid>
                 </div>
               </Card>
-            ))}
+            ))
+          ) : (
+            <p>No Job Followups with {company}</p>
```

Comment:
Looks nice but lets try to use `<Typography/>` only. Add fonts to the theme in App.js